cli/requirements.txt

```diff
@@ -19,6 +19,8 @@ aws-cdk.aws-logs~=1.116
 aws-cdk.aws-route53~=1.116
 aws-cdk.aws-ssm~=1.116
 aws-cdk.aws-sqs~=1.116
+aws-cdk.aws-autoscaling~=1.116
+aws-cdk.aws-cloudformation~=1.116
```

Comment:
do we need to update deps in setup.py too?

Reply:
done